addons/website/views/snippets/snippets.xml

```diff
@@ -802,6 +804,11 @@
             <we-button data-customize-website-views="website.header_language_selector_flag, website.header_language_selector_no_text">Flag</we-button>
             <we-button data-customize-website-views="website.header_language_selector_flag">Flag and Text</we-button>
         </we-select>
+        <we-select string="Logo type" data-reload="/" data-dependencies="option_header_brand_none">
+            <we-button data-customize-website-views="" data-name="option_header_brand_none">None</we-button>
+            <we-button data-customize-website-views="website.option_header_brand_name">Text</we-button>
+            <we-button data-customize-website-views="website.option_header_brand_logo">Image</we-button>
+        </we-select>
```

Comment:
Kinda annoying to have this duplicated, make a dedicated template (and parametrable for dependencies and string) that you can t-call at the two places